docs/dev-guide/3.0-development/data-modeling.rst

```diff
@@ -4,13 +4,13 @@ Data Modeling
 Introduction
 ^^^^^^^^^^^^
 
-The pulp 3 data modeling effort is not just a translation or porting effort but instead
-an effort to build pulp 3 using pulp 2.  Remodeling will include changes to leverage the
-relational capabilities of postgres but also to improve upon the pulp 2 model.
+The Pulp 3 data modeling effort is not just a translation or porting effort but instead
+an effort to build Pulp 3 using Pulp 2.  Remodeling will include changes to leverage the
+relational capabilities of postgres but also to improve upon the Pulp 2 model.
 
-When creating each pulp 3 model object, consider the following:
+When creating each Pulp 3 model object, consider the following:
 
-- Understand what the pulp 2 model (collection) stores and how the information
+- Understand what the Pulp 2 model (collection) stores and how the information
   is used by pulp.
```

Comment:
Not sure it matters.  I actually made more changes here WRT case than I intended doing global replaces.  If folks feel strongly I'll do what's needed here.